docs/en/ingest-management/troubleshooting/troubleshooting.asciidoc

```diff
@@ -650,11 +650,28 @@ To work around this problem, you can reset the {ecloud} agent policy with an API
 curl -u elastic:<password> --request POST \
   --url <kibana_url>/internal/fleet/reset_preconfigured_agent_policies/policy-elastic-agent-on-cloud \
   --header 'Content-Type: application/json' \
-  --header 'kbn-xsrf: xyz' \
+  --header 'kbn-xsrf: xyz'
 ----
 
 [discrete]
 [[pgp-key-download-fail]]
 == Air-gapped {agent} upgrade can fail due to an inaccessible PGP key
 
 In versions 8.9 and above, an {agent} upgrade may fail when the upgrader can't access a PGP key required to verify the binary signature. For details and a workaround, refer to the link:https://www.elastic.co/guide/en/fleet/8.9/release-notes-8.9.0.html#known-issue-3375[PGP key download fails in an air-gapped environment] known issue in the version 8.9.0 Release Notes or to the link:https://github.com/elastic/elastic-agent/blob/main/docs/pgp-workaround.md[workaround documentation] in the elastic-agent GitHub repository.
+
+[discrete]
+[[fleet-server-integration-removed]]
+== {agents} unable to connect after removing {fleet-server} integration
+
+When you use {fleet}-managed {agent}, at least one {agent} is running the link:https://docs.elastic.co/integrations/fleet_server[{fleet-server} integration]. In case the policy containing this integration is accidentally removed from {agent], all other agents will not be able to be managed. However, the {agents} will continue to send data to their configured output.
```

Comment:
Typo:
```suggestion
When you use {fleet}-managed {agent}, at least one {agent} is running the link:https://docs.elastic.co/integrations/fleet_server[{fleet-server} integration]. In case the policy containing this integration is accidentally removed from {agent}, all other agents will not be able to be managed. However, the {agents} will continue to send data to their configured output.
```

Reply:
Fixed in [6bd21ce](https://github.com/elastic/ingest-docs/pull/815/commits/6bd21cee444aca00ce0ddb2d51d0efbaeb29730e)